data-science-onramp/data-ingestion/setup.py

```diff
@@ -1,17 +1,37 @@
 import random
 import sys
+import pandas as pd
 
 from google.cloud import bigquery
 from py4j.protocol import Py4JJavaError
 from pyspark.sql import SparkSession
 from pyspark.sql.functions import expr, UserDefinedFunction, when
-from pyspark.sql.types import StringType
+from pyspark.sql.types import FloatType, StringType, StructField, StructType
 
 
 BUCKET_NAME = sys.argv[1]
 TABLE = "bigquery-public-data.new_york_citibike.citibike_trips"
-RAW_DATASET_NAME = "new_york_citibike_trips"
-RAW_TABLE_NAME = "RAW_DATA"
+DATASET_NAME = "data_science_onramp"
+RAW_TABLE_NAME = "new_york_citibike_trips"
+EXTERNAL_DATASETS = {
```

Comment:
Instead of having an external datasets dictionary, can we just call it datasets and group in the citibike one too? Asking because I see that there is some repetitive code below (e.g. `if upload: write_to_bigquery ...`)? We might need to make some tweaks if we do this because it does not have a URL so not sure if it's worth it. I'm curious to see what everyone thinks!